src/cloud-element-templates/util/propertyUtil.js

```diff
@@ -154,7 +155,19 @@ export function getPropertyValue(element, property, scope) {
       const zeebeProperty = findZeebeProperty(zeebeProperties, binding);
 
       if (zeebeProperty) {
-        return zeebeProperty.get('value');
+        const value = zeebeProperty.get('value');
```

Comment:
So we only do the casting for `zeebe:Property`? Why not for others? For example if a regular `property` would be a boolean shouldn't we also do the casting there?

Reply:
My goal was to fix the reported issue, but indeed we allow `Boolean` in multiple places, so we should do casting for each one.

Reply:
Ok, let merge this and create follow-up issue so we don't forget about it. Sounds good?

Reply:
OK

Reply:
Though I'm pretty close to have a general fix.